src/base/bittorrent/trackerentry.cpp

```diff
@@ -57,19 +57,20 @@ int TrackerEntry::tier() const
     return nativeEntry().tier;
 }
 
-TrackerEntry::Status TrackerEntry::status() const
+TrackerEntry::Status TrackerEntry::status(const InfoHashFormats hashVersions) const
```

Comment:
Now such methods can be called with empty set of formats. Do you handle this case?

Reply:
Not really, none of the values in `enum Status` are suitable here. Maybe use `q_assert()`?

Reply:
>Maybe use `q_assert()`?

Yes, at least.
Although you are unlikely to pass an incorrect parameter here by doing it directly, but if this is used in more complex logic, when flags are built based on some other data or conditions, then some error may cause it to be left empty here. So there must be something that wakes up to indicate such an error.